packages/bsky/src/api/app/bsky/notification/listNotifications.ts

```diff
@@ -90,10 +90,10 @@ export default function (server: Server, ctx: AppContext) {
         labelService.getLabelsForUris(recordUris),
       ])
 
-      const notifications = notifs.map((notif, i) => ({
+      const notifications = notifs.map((notif) => ({
         uri: notif.uri,
         cid: notif.cid,
-        author: authors[i],
+        author: authors[notif.authorDid],
```

Comment:
Similar here, we might consider skipping notifications that don't have an author.

Totally understandable if we want to do these in a separate pass, since I wouldn't anticipate an actual regression.

Reply:
yup 👍